package.json

```diff
@@ -32,7 +32,9 @@
     "test:all": "npm run test:node && npm run test:browser",
     "test:node": "mocha --retries 2 --reporter spec --require intelli-espower-loader --exit",
     "test:browser": "testem",
-    "prepublish": "node -e \"if(process.env.npm_package_version!==require('./lib/VERSION')){console.error('The pacakge.json version is not matching to ./lib/VERSION.js');process.exit(1)}\""
+    "prepublish": "node -e \"if(process.env.npm_package_version!==require('./lib/VERSION')){console.error('The pacakge.json version is not matching to ./lib/VERSION.js');process.exit(1)}\"",
+    "snyk-protect": "snyk protect",
+    "prepare": "npm run snyk-protect"
```

Comment:
Will this fail a publish if `snyk protect` finds anything? I'm not sure that is a good thing as some vulnerabilities are addressed at different times.

Reply:
What's interesting with the way snyk sets this up, an `npm install` in a /downstream/ project that includes `jsforce` as a dependency will execute this `prepare` statement and monkey-patch lodash (from the .snyk policy file) in the downstream project's npm_modules/ dir.

From what I can tell, the `protect` phase does not check for new vulnerabilities or error out if new vulnerabilities are found.

References:
- https://docs.npmjs.com/misc/scripts#prepublish-and-prepare
- https://support.snyk.io/hc/en-us/articles/360000920818

Reply:
Removed snyk bits since lodash update .19 is now published.